UserInputValidator.cs

```diff
@@ -2,21 +2,15 @@ namespace TictactoeVer2
 {
     public class UserInputValidator
     {
-        public bool IsBoardSizeValid(string input)
+        public bool TryParseBoardSize(string input, out int size)
         {
-            var size = ParseBoardSize(input);
-            
-            return IsWithinRange(size);
+            SetInputToDefaultSizeIfBlank(ref input);
```

Comment:
Surely we could simply return the default size if blank, rather than handing it to the parse and validation logic below?